vertical-pod-autoscaler/pkg/updater/eviction/pods_eviction_restriction_test.go

```diff
@@ -642,7 +642,7 @@ func TestEvictEmitEvent(t *testing.T) {
 			mockRecorder.On("Event", mock.Anything, apiv1.EventTypeNormal, "EvictedByVPA", mock.Anything).Return()
 			mockRecorder.On("Event", mock.Anything, apiv1.EventTypeNormal, "EvictedPod", mock.Anything).Return()
 
-			eviction.Evict(p.pod, testCase.vpa, mockRecorder)
+			_ = eviction.Evict(p.pod, testCase.vpa, mockRecorder)
```

Comment:
@adrianmoisey error check here cause a failure (https://github.com/kubernetes/autoscaler/actions/runs/12082981510/job/33695187736) so I ignore that for now.

Reply:
sure. In this PR?

Reply:
I'm not too fussed, your choice

Reply:
👍